test/jdk/java/awt/Mixing/AWT_Mixing/SimpleOverlappingTestBase.java

```diff
@@ -55,6 +67,9 @@ protected SimpleOverlappingTestBase(boolean defaultClickValidation) {
         this.useDefaultClickValidation = defaultClickValidation;
     }
 
+    protected boolean isMultiFramesTest(){
+        return true;
+    }
     public SimpleOverlappingTestBase() {
```

Comment:
```suggestion
    }

    public SimpleOverlappingTestBase() {
```

Ensure there's a blank line around method declarations. (For modified / added methods.)

Reply:
Done.